flintrock/config.yaml.template

```diff
@@ -5,6 +5,9 @@ services:
     # git-repository:  # optional; defaults to https://github.com/apache/spark
   hdfs:
     version: 2.7.1
+    # download-source: # optional; defaults to
+    # http://www.apache.org/dyn/closer.lua/hadoop/common/hadoop-{v}/hadoop-{v}.tar.gz?as_json
```

Comment:
Since they are lengthy, let's move all the comment lines for this option to right above the `# download-source: ` line.